test/parallel/test-crypto-stream.js

```diff
@@ -72,7 +72,14 @@ const decipher = crypto.createDecipheriv('aes-128-cbc', badkey, iv);
 
 cipher.pipe(decipher)
   .on('error', common.mustCall(function end(err) {
-    assert(/bad decrypt/.test(err));
+    assert.throws(() => { throw err; },
+      {
+        message: /bad decrypt/,
+        function: 'EVP_DecryptFinal_ex',
+        library: 'digital envelope routines',
+        reason: 'bad decrypt',
+      }
+    );
```

Comment:
Instead of using this construct it's possible to replace the whole callback with:

```js
.on('error', common.expectsError({
  message: /bad decrypt/,
  function: '...',
  ...
}))
```